src/components/banner/index.tsx

```diff
@@ -11,9 +11,10 @@ import styles from './banner.module.scss';
 // we put a more robust solution in place.
 //
 const SHOW_BANNER = true;
-const BANNER_TEXT = 'Join Our Live Demo: Intro to Sentry & Codecov on June 25';
+const BANNER_TEXT =
+  'Behind the Code: A Conversation With Backend Experts" featuring CEOs of Laravel, Prisma, and Supabase.';
```

Comment:
```suggestion
  'Behind the Code: A Conversation With Backend Experts featuring CEOs of Laravel, Prisma, and Supabase.';
```
Looks like there's an extra quotation mark here